code.js

```diff
@@ -1,3 +1,15 @@
 are_isomorphic(graph1, graph2) {
-    return undefined;
+    if (graph1.vertices.length !== graph2.vertices.length || graph1.edges.length !== graph2.edges.length) {
+    return false;
 }
+    let adjMatrix1 = graph1.getAdjacencyMatrix();
+    let adjMatrix2 = graph2.getAdjacencyMatrix();
+
+    let perms = getAllPermutations(graph2.vertices)
```

Comment:
Sorry, I'm missing something here -- where do all the functions you're using come from?

Reply:
https://www.mathworks.com/help/simbio/ref/getadjacencymatrix.html

https://igraph.org/r/html/1.2.6/isomorphic.html

https://docs.seqan.de/seqan1/1.2/FUNCTION.get_Adjacency_Matrix.html

Reading and googling stuff

Reply:
But these functions aren't defined anywhere in your code.

Reply:
I just changed it so it would check the connectivity and pass the test code. I'll add the functions in